lib/tasks/remove_deleted_roster_entries.rake

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+task remove_deleted_roster_entries: :environment do
+  puts "Removing roster entries connected to deleted rosters"
+
+  BATCH_SIZE = 100
+
+  roster_entries_deleted = 0
+  batch_num = 1
+
+  non_existing_rosters = RosterEntry.where.not(roster_id: Roster.pluck(:id)).pluck(:roster_id).uniq
+
+  non_existing_rosters.each do |roster_id|
+    roster_entries_to_delete = RosterEntry.where(roster_id: roster_id)
```

Comment:
this will find all the roster entries and generate an array of ActiveRecord objects, we can fix that by not assigning the output to `roster_entries_to_delete` and chain the query with find in batches. let me know if this works.

Reply:
Ahhh that makes more sense